docs/UserGuide.md

```diff
@@ -126,118 +176,294 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
+
+* `PARTICIPATION` **must be a positive integer** from 0 to 500.
+* `LESSON_INDEX` refers to the index number shown in the displayed list of tasks.
+* `STUDENT_INDEX` refers to the index number shown in the student list for the specified lesson. (use `listlesson` to view the student list for lessons)
+* `LESSON_INDEX` and `STUDENT_INDEX` **must be a positive integer** 1, 2, 3, …​
+
+Example:
+* `addpart p/100 li/1 si/2` sets participation for the 2nd student from the 1st lesson to be 100.
+
+## Saving the data
 
 JARVIS data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
-### Editing the data file
+## Editing the data file
 
-JARVIS data is saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+JARVIS data is saved in 3 JSON files:
+1. `[JAR file location]/data/studentbook.json` for student data
+2. `[JAR file location]/data/taskbook.json` for task data
+3. `[JAR file location]/data/lessonbook.json` for lesson data
+
+Advanced users are welcome to update data directly by editing the data files.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
 If your changes to the data file makes its format invalid, JARVIS will discard all data and start with an empty data file at the next run.
 </div>
 
-### Archiving data files `[coming in v2.0]`
-
-_Details coming soon ..._
-
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous JARVIS home folder.
+**A**: Install the app in the other computer and overwrite the empty data files it creates with the files that contains the data of your previous JARVIS home folder.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
 
-Action | Format, Examples
---------|------------------
-**Add** | `add s/NAME`, `add t/TASK_DESC [d/DATE]` <br> e.g., `add s/John Tan`, `add t/Consultation d/12-09-2023`
-**Clear** | `clear`
-**Delete** | `delete s/STUDENT_INDEX` or `delete t/TASK_INDEX` <br> e.g., `delete s/3` or `delete t/2`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list s` or `list t`
-**Mark** | `mark t/TASK_INDEX` <br> e.g., `mark t/2`
-**Unmark** | `unmark t/TASK_INDEX` <br> e.g., `unmark t/2`
-**Mastery Check**| `mc MC_NUM s/STUDENT_INDEX r/RESULT` <br> e.g., `mc 1 s/1 r/PASS`
-**Help** | `help`
+Action | Format                                                           | Examples
```

Comment:
It seems like in the original AB3 UG this table is sorted alphabetically, but I just put it according to the order above. Not sure if it has to be sorted alphabetically.